db/migrate/20170116210702_add_manager_ref_to_authentications.rb

```diff
@@ -0,0 +1,5 @@
+class AddManagerRefToAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    add_column :authentications, :manager_ref, :string
```

Comment:
in other providers, we are using polymorphic :as => :resource, so manager_ref was not needed